app/services/import_actuals.rb

```diff
@@ -88,8 +89,12 @@ def create_actual
       attrs = @converter.to_h
       assign_default_values(attrs)
 
-      creator = CreateActual.new(activity: @activity, report: @report)
-      result = creator.call(attributes: attrs)
+      creator = if @converter.actual_class == Refund
+        CreateRefund.new(activity: @activity, report: @report, user: @uploader)
```

Comment:
Not directly related to this line, but now we're creating refunds, does the code/naming of the method on [L58](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/f914dc35b7dd7434ea6358f9a3c1de1b7d2a1761/app/services/import_actuals.rb#L58) still make sense?

Reply:
`zero_value?` does feel like a more reasonable name at this point